exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -314,6 +370,31 @@ public void deleteAllOptions(OptionType type) {
     }
   }
 
+  public void populateDefualtValues(Map<String, OptionValidator> VALIDATORS) {
+
+    // populate the options from the config
+    final Map<String, OptionValidator> tmp = new HashMap<>();
```

Comment:
No need to create a new map. Setting the default value does not change the key, so you can just update the fields of the value. (That is, we are not even replacing the value; we are just altering a field in the value.)